nnpackage/schema/circle_schema.fbs

```diff
@@ -360,6 +360,7 @@ enum BuiltinOperator : ubyte {
   DENSIFY = 124,
   SEGMENT_SUM = 125,
   BATCH_MATMUL = 126,
+  LQ_FULLY_CONNECTED = 200,
```

Comment:
May better to change this num?

Reply:
Yes.
If you want to merge it in master branch we need to discuss it with maintainers.
200 is definitely not the best value, because it could be taken by tflite in future releases and we want to preserve compatibility with it.